apps/site/pages/en/blog/events/collab-summit-2025-paris.md

```diff
@@ -0,0 +1,96 @@
+---
+date: '2025-05-22T00:00:00.000Z'
+category: events
+title: 'Trip report: Node.js collaboration summit (2025 Paris)'
+layout: blog-post
+author: joyeecheung
```

Comment:
I am not sure whether this supports multiple authors, but this is co-authored by @legendecas

Reply:
It does! Comma separated :) `,`

Reply:
Thanks, done.